.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "stage": 1
```

Comment:
Needed to support `{scaffold, ...opts}` (line 25 of index.js) -  object destructuring.